lib/output.ts

```diff
@@ -69,10 +69,26 @@ export function mismatchingVersionsFixedToOutput(
     throw new Error('No fixes to output.');
   }
 
-  const dependencies = mismatchingDependencyVersions
-    .map((mismatchingVersion) => mismatchingVersion.dependency)
-    .sort();
-  return `Fixed versions for ${dependencies.length} ${
-    dependencies.length === 1 ? 'dependency' : 'dependencies'
-  }: ${dependencies.join(', ')}`;
+  const dependenciesAndFixedVersions = mismatchingDependencyVersions
+    .flatMap((mismatchingVersion) => {
+      let version;
+      try {
+        version = getHighestVersion(
+          mismatchingVersion.versions.map((v) => v.version)
+        );
+      } catch {
+        return []; // Ignore this dependency since unable to get the version that we would have fixed it to.
+      }
+      return {
+        dependency: mismatchingVersion.dependency,
+        version,
+      };
+    })
+    .sort((a, b) => a.dependency.localeCompare(b.dependency));
+
+  return `Fixed versions for ${dependenciesAndFixedVersions.length} ${
+    dependenciesAndFixedVersions.length === 1 ? 'dependency' : 'dependencies'
+  }: ${dependenciesAndFixedVersions
+    .map((object) => `${object.dependency} to ${object.version}`)
```

Comment:
```suggestion
    .map((object) => `${object.dependency}@${object.version}`)
```

Reply:
I considered that but then we end up with stuff like `@types/qunit@1.2.3` for nested dependencies...If there's prior art for using that format let me know.

Reply:
`yarn upgrade @types/qunit@1.2.3` is standard yarn syntax.

Reply:
Fair enough, updated.